src/Components/WebAssembly/Sdk/src/targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets

```diff
@@ -631,6 +631,13 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ItemGroup>
   </Target>
 
+  <Target Name="_FailIfReferencingAspNetCoreApp" BeforeTargets="ResolveRuntimePackAssets">
+    <Error
+      Code="BLZ3001"
```

Comment:
Mostly for my own curiosity, is this code just made up?  BLZ = blazor and 3001, in this file I see another RAZORSDK1007 code, is there

Reply:
Hmm, I forgot we updated SDK diagnostic codes. I can change this BlazorSDK1001 or something. But yes, it's made up.